js/index.js

```diff
@@ -6,19 +6,13 @@
   var exceptions = {
     // for the Now tab
     "applicationmanager.gov/application.aspx": "https://applicationmanager.gov",
-    "forecast.weather.gov/mapclick.php": "http://forecast.weather.gov/",
+    "forecast.weather.gov/mapclick.php": "http://www.weather.gov/",
```

Comment:
Not sure if `http://forecast.weather.gov` redirected before, but it just redirects to weather.gov now anyway.